src/item.cpp

```diff
@@ -8203,6 +8205,33 @@ cata::optional<tripoint> item::get_cable_target( player *p, const tripoint &pos
 
 bool item::process_cable( player *p, const tripoint &pos )
 {
+    if( p == nullptr ) {
+        reset_cable( p );
+    }
+    std::string state = get_var( "state" );
+    if( state == "solar_pack_link" ) {
+        if( !p->has_item( *this ) || ( !p->is_wearing( "solarpack_on" ) ||
```

Comment:
This may get called with `p` being null.

Reply:
`if( p == nullptr ) { reset_cable( p ); }` is checked at the very begining of the `process_cable()` 
I'll add `return false;` that should bail out from there.